x/posts/types/codec.go

```diff
@@ -17,21 +18,21 @@ func RegisterLegacyAminoCodec(cdc *codec.LegacyAmino) {
 	cdc.RegisterConcrete(&Poll{}, "desmos/Poll", nil)
 	cdc.RegisterConcrete(&Media{}, "desmos/Media", nil)
 
-	cdc.RegisterConcrete(&MsgCreatePost{}, "desmos/MsgCreatePost", nil)
-	cdc.RegisterConcrete(&MsgEditPost{}, "desmos/MsgEditPost", nil)
-	cdc.RegisterConcrete(&MsgAddPostAttachment{}, "desmos/MsgAddPostAttachment", nil)
-	cdc.RegisterConcrete(&MsgRemovePostAttachment{}, "desmos/MsgRemovePostAttachment", nil)
-	cdc.RegisterConcrete(&MsgDeletePost{}, "desmos/MsgDeletePost", nil)
-	cdc.RegisterConcrete(&MsgAnswerPoll{}, "desmos/MsgAnswerPoll", nil)
-	cdc.RegisterConcrete(&MsgMovePost{}, "desmos/MsgMovePost", nil)
+	legacy.RegisterAminoMsg(cdc, &MsgCreatePost{}, "desmos/MsgCreatePost")
+	legacy.RegisterAminoMsg(cdc, &MsgEditPost{}, "desmos/MsgEditPost")
+	legacy.RegisterAminoMsg(cdc, &MsgAddPostAttachment{}, "desmos/MsgAddPostAttachment")
+	legacy.RegisterAminoMsg(cdc, &MsgRemovePostAttachment{}, "desmos/MsgRemovePostAttachment")
+	legacy.RegisterAminoMsg(cdc, &MsgDeletePost{}, "desmos/MsgDeletePost")
+	legacy.RegisterAminoMsg(cdc, &MsgAnswerPoll{}, "desmos/MsgAnswerPoll")
+	legacy.RegisterAminoMsg(cdc, &MsgMovePost{}, "desmos/MsgMovePost")
 
-	cdc.Amino.RegisterConcrete(&MsgRequestPostOwnerTransfer{}, "desmos/MsgRequestPostOwnerTransfer", nil)
-	cdc.Amino.RegisterConcrete(&MsgCancelPostOwnerTransferRequest{}, "desmos/MsgCancelPostOwnerTransferRequest", nil)
-	cdc.Amino.RegisterConcrete(&MsgAcceptPostOwnerTransferRequest{}, "desmos/MsgAcceptPostOwnerTransferRequest", nil)
-	cdc.Amino.RegisterConcrete(&MsgRefusePostOwnerTransferRequest{}, "desmos/MsgRefusePostOwnerTransferRequest", nil)
+	legacy.RegisterAminoMsg(cdc, &MsgRequestPostOwnerTransfer{}, "desmos/MsgRequestPostOwnerTransfer")
+	legacy.RegisterAminoMsg(cdc, &MsgCancelPostOwnerTransferRequest{}, "desmos/MsgCancelPostOwnerTransfer")
+	legacy.RegisterAminoMsg(cdc, &MsgAcceptPostOwnerTransferRequest{}, "desmos/MsgAcceptPostOwnerTransfer")
+	legacy.RegisterAminoMsg(cdc, &MsgRefusePostOwnerTransferRequest{}, "desmos/MsgRefusePostOwnerTransfer")
```

Comment:
amino name of OwnerTransferRequest msg are too long, so remove Request from them.